web/src/components/input/InputWithTags.tsx

```diff
@@ -421,7 +437,11 @@ export default function InputWithTags({
         ? t("button.yes", { ns: "common" })
         : t("button.no", { ns: "common" });
     } else if (filterType === "labels") {
-      return getTranslatedLabel(String(filterValues));
+      const value = String(filterValues);
+      return getTranslatedLabel(
+        value,
+        allAudioListenLabels.includes(value) ? "audio" : "object",
```

Comment:
Same as other areas it would be better to avoid this, or at least create a mapping in usememo so it isn't rerun every time